src/Microsoft.Graph.Cli.Core/IO/GraphCliClientFactory.cs

```diff
@@ -1,12 +1,28 @@
 using System.Linq;
 using System.Collections.Generic;
 using System.Net.Http;
+using System;
+using Microsoft.Kiota.Http.HttpClientLibrary.Middleware;
+using System.Net.Http.Headers;
 
 namespace Microsoft.Graph.Cli.Core.IO;
 
 public class GraphCliClientFactory
 {
-    public static IEnumerable<DelegatingHandler> GetDefaultMiddlewaresWithOptions(GraphClientOptions options) => GraphClientFactory.CreateDefaultHandlers(options);
+    /// Microsoft Graph service national cloud endpoints
+    private static readonly Dictionary<string, string> cloudList = new Dictionary<string, string>
```

Comment:
```suggestion
    private static readonly Dictionary<string, string> cloudList = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase)
```

Reply:
Is this necessary? The keys are constant, and the dictionary is read only....